src/client/common/platform/pathUtils.ts

```diff
@@ -48,6 +48,11 @@ export class PathUtils implements IPathUtils {
     }
 
     public getDisplayName(pathValue: string, cwd?: string): string {
+        // Paths on windows can either contain \ or / Both work.
+        // Thus, C:\Python.exe is the same as C:/Python.exe
+        // If we're on windows ensure we convert the / in pathValue to \.
+        // For cases like here https://github.com/microsoft/vscode-jupyter/issues/399
+        pathValue = this.isWindows ? pathValue.replace(/\//g, '\\') : pathValue;
```

Comment:
@IanMatthewHuff @rchiodo 
Added this change to account for duplicates seen in https://github.com/microsoft/vscode-jupyter/issues/399
Not sure this is the cause of the duplicates, but would like to fix this as well

Reply:
Basically on windows all paths will always be displayed (display only) with a `\` instead of `/`.

Reply:
Seems legit. Would expect to see windows path names looking like that.